interface-definitions/vpp.xml.in

```diff
@@ -643,17 +643,17 @@
                 </leafNode>
               </children>
             </node>
-            <leafNode name="route-no-paths">
+            <leafNode name="disable-route-no-paths">
               <properties>
-                <help>Route no paths</help>
+                <help>Disable route no paths option</help>
```

Comment:
I can't find the docs for that option in the VPP manual and from this help string, I'd never be able to guess what the option actually does.

Do you think it's possible to make it more descriptive?

If you point me at the docs, I can also think about it, once I know what it does. :)

Reply:
> I can't find the docs for that option in the VPP manual and from this help string, I'd never be able to guess what the option actually does.

It was added from the [patch](https://github.com/vyos/vyos-vpp-patches/blob/current/patches/vpp/0002-linux-cp-Added-routing-for-prefixes-with-no-paths-av.patch)

The most suitable option `learn-kernel-routes  < enable|disable >`

What do you think about it?

Reply:
`learn-kernel-routes` sounds good to me! Although we made an agreement long ago that we do not create nodes with arguments like `true/false`, `enable/disable`, etc.

If you think the most sensible default is to have the dataplane learn kernel routes, then we should have a valueless node like `disable-learning-kernel-routes` (or similar, e.g., `ignore-kernel-routes`). 

If you think the default should be not to learn then, we should add a valueless node `learn-kernel-routes`.

Reply:
Done!
```
set vpp settings lcp ignore-kernel-routes
```